velox/functions/sparksql/Size.cpp

```diff
@@ -49,14 +53,30 @@ struct Size {
     return true;
   }
 
+  template <typename TInput>
+  FOLLY_ALWAYS_INLINE bool callNullable(
+      int32_t& out,
+      const TInput* input,
+      const bool* /*legacySizeOfNull*/) {
+    if (input == nullptr) {
+      if (legacySizeOfNull_) {
+        out = -1;
+        return true;
+      }
+      return false;
+    }
+    out = input->size();
+    return true;
+  }
+
  private:
   bool legacySizeOfNull_;
```

Comment:
Let's add some comments for this variable as the config and its description are removed.

> // If true, returns -1 for null input. Otherwise, returns null for null input.